src/client/src/App/Credit/SellSide/SellSideTradeTicket/SellSideTradeTicketCore.tsx

```diff
@@ -123,6 +123,7 @@ export const SellSideTradeTicketTicketCore = ({
           quoteState={quote?.state ?? QuoteState.Pending}
         />
         <SellSideTradeTicketParameters
+          rfqId={rfqId}
```

Comment:
Very minor, this could have a more explicit name like selectedRfqId, I had to trace it up through the components to understand what exactly it was.